tests/cypress/support/pro/commands.js

```diff
@@ -32,6 +32,11 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.on("uncaught:exception", (err, runnable) => {
+  // returning false here prevents Cypress from failing the test
+  return false;
+});
+
```

Comment:
This will make cypress ignore all uncaught JS runtime exceptions, are we sure this is what we want? If possible, it might be better to identify and fix the actual error that's thrown instead.

Reply:
If we want to workaround a specific error, do you think you could narrow it down as explained here: https://docs.cypress.io/api/cypress-api/catalog-of-events#To-conditionally-turn-off-uncaught-exception-handling-for-a-certain-error

Reply:
Thanks @petermakowski 

I will narrow it down to a specific error, then. I am not sure why this typing error happens on cypress. It does not pop up on prod/demos.